paymentsheet/src/main/java/com/stripe/android/paymentsheet/repositories/CustomerApiRepository.kt

```diff
@@ -96,7 +96,6 @@ internal class CustomerApiRepository @Inject constructor(
                         Wallet.Type.ApplePay,
                         Wallet.Type.GooglePay,
                         Wallet.Type.SamsungPay,
-                        Wallet.Type.Link,
```

Comment:
Do we need to dedupe these? I think that's the reason we added this to begin with.

Reply:
Yeah we will need to dedupe them. Right now, web is not deduping so this is acceptable behavior for now but we should dedupe them for next release.